README.md

```diff
@@ -6,7 +6,7 @@ GOV.UK stack.
 ## How to use
 
 ```shell
-rails new APP_NAME -m http://raw.github.com/govuk-rails-app-template/master/template.rb
+rails new APP_NAME --skip-test-unit --skip-spring -m govuk-rails-app-template/template.rb
```

Comment:
Updated with `--no-javascript` and RBENV prefix. 

I think we'll ultimately end up with either separate templates for the different types of app, or the template will become interactive, e.g.

```ruby
if yes?("is this a frontend application?")
  # setup with frontend stuff, e.g. slimmer, etc
elsif yes?("is this an admin application?")
  # setup with admin stuff, e.g. govuk_admin_template, gds-sso, etc
else
  # it's something else, like authenticating-proxy
end
```